testproject/Assets/Tests/Manual/SceneTransitioningAdditive/NetworkManagerMonitor.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using Unity.Netcode;
+
+public class NetworkManagerMonitor : MonoBehaviour
+{
+    // Start is called before the first frame update
+    private void Start()
```

Comment:
So this is just when we add this to a GameObject with a NetworkManager it will destroy all other listening NetworkManagers on Start right?

Reply:
Yep!